<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testLightController</class>
 <widget class="QDialog" name="testLightController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Light Controller</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="LightCntlr">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Light Controller:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="lightCntlrComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelGroups">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Groups:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCntlrNum">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>!0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="groupsLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labelCommand">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Command:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="commandsComboBox">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>120</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="TimeOn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time On:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeOnLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="TimeOff">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time Off:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeOffLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="numFlashes">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Number of Flashes:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numFlashesLineEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="TimeDelay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time Delay:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="timeDelayLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="Color">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="colorLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="sideColor">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Side Color:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sideColorLineEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="Color_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Color Map:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="colorMapLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Command</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>testLightController</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>testLightController</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
